app/assets/javascripts/admin/addon/components/admin-config-areas/about.gjs

```diff
@@ -1,14 +1,44 @@
 import Component from "@glimmer/component";
+import i18n from "discourse-common/helpers/i18n";
+import AdminConfigAreaCard from "admin/components/admin-config-area-card";
+import AdminConfigAreasAboutContactInformation from "admin/components/admin-config-area-cards/about/contact-information";
+import AdminConfigAreasAboutGeneralSettings from "admin/components/admin-config-area-cards/about/general-settings";
+import AdminConfigAreasAboutYourOrganization from "admin/components/admin-config-area-cards/about/your-organization";
 
 export default class AdminConfigAreasAbout extends Component {
-  cards = [1, 2, 3];
+  saveCallback() {
+    // eslint-disable-next-line no-console
+    console.log("save callback");
+  }
 
   <template>
     <div class="admin-config-area">
+      <h2>{{i18n "admin.config_areas.about.header"}}</h2>
```

Comment:
We should have breadcrumbs above the title using `DBreadcrumbsContainer`, **however** the API for that is changing a bit so that can be done in a follow up PR. Likely we are going to need a generic `AdminPageHeader` component that we can easily add title/description/breadcrumbs/tabs to as the UI guidelines develop.

Reply:
Yup, I'll do this in a later PR.